governance.md

```diff
@@ -1,14 +1,32 @@
-# CDEvents Bootstrap Governance
+# CDEvents Governance
 
-The initial bootstrap committee will consist of 5 individuals who are core stakeholders and/or contributors.
+The CDEvents Governance committee is the governing body of the CDEvents open source project. It's an elected group that represents the contributors to the project, and has an oversight on governance and technical matters.
 
-Members are (in alphabetical order):
 
-* Emil Bäckmark, [@e-backmark-ericsson](https://github.com/e-backmark-ericsson), Ericsson
-* Andrea Frittoli, [@afrittoli](https://github.com/afrittoli), IBM
-* Mattias Linnér, [@m-linner-ericsson](https://github.com/m-linner-ericsson), Ericsson
-* Erik Sternerson [@erkist](https://github.com/erkist), doWhile
-* Steve Taylor [@sbtaylor15](https://github.com/sbtaylor15), [DeployHub](https://www.deployhub.com) / [Ortelius OS](https://ortelius.io)
+- [CDEvents Governance Committee](#CDEvents-governance-committee)
+- [Governance Facilitator Role (optional)](#governance-facilitator-role-optional)
+- [Maximum Representation](#maximum-representation)
+- [Committee Responsibilities and Deliverables](#committee-responsibilities-and-deliverables)
+- [Governance Meetings and Decision-Making Process](#governance-meetings-and-decision-making-process)
+- [Elections](#elections)
+- [Permissions and access](#permissions-and-access)
+- [Governing board leave policy](#governing-board-leave-policy)
+
+## CDEvents Governance Committee
+
+The CDEvents governance committee consists of five elected individuals. The five seats are held for two year terms, staggered by one year: every year either two or three of the seats are up for election.
```

Comment:
Great question, I thought about it too, but didn't include an answer yet.
I would like to aim for the 5 members to hopefully get more companies and end users represented, but we might want to have a fallback policy in case it doesn't happen.

Reply:
Yes, I think it's good to have something written about that scenario, so we don't end up there and don't know how to proceed.